modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/filters/Filter.js

```diff
@@ -74,11 +94,24 @@ const Filter = ({moduleURL, type, ...otherProps}) => {
 
 	useEffect(() => {
 		if (moduleURL) {
-			getComponentByModuleURL(moduleURL).then((FetchedComponent) =>
-				setComponent(() => FetchedComponent)
-			);
+			if (type === 'client-extension') {
+				const getModule = async () => {
+					const cetModule = await import(
+						/* webpackIgnore: true */ moduleURL
+					);
+
+					setComponent(() => cetModule['default']);
+				};
+
+				getModule();
+			}
+			else {
+				getComponentByModuleURL(moduleURL).then((FetchedComponent) =>
+					setComponent(() => FetchedComponent)
+				);
+			}
```

Comment:
It is similar because `getCellRendererByURL` ends up calling `getComponentByModuleURL`, so we are already sharing a generic variant. `getCellRendererByURL` is just written with some specifics for cell renderer